src/store/modules/wallet.js

```diff
@@ -575,10 +578,21 @@ const actions = {
     const txHash = categorizedEvent.tx_hash;
     const event = { ...categorizedEvent };
     let memo;
+    let batchSendList = null;
```

Comment:
Can we just use `event.tx_hash` to group events in `processEvents` without calling an extra API?